make/hotspot/lib/JvmOverrideFiles.gmk

```diff
@@ -55,12 +55,19 @@ ifeq ($(call isTargetOs, linux), true)
   BUILD_LIBJVM_sharedRuntimeTrig.cpp_CXXFLAGS := -DNO_PCH $(FDLIBM_CFLAGS) $(LIBJVM_FDLIBM_COPY_OPT_FLAG)
   BUILD_LIBJVM_sharedRuntimeTrans.cpp_CXXFLAGS := -DNO_PCH $(FDLIBM_CFLAGS) $(LIBJVM_FDLIBM_COPY_OPT_FLAG)
 
+  ifeq ($(call isTargetCpu, ppc64le)+$(TOOLCHAIN_TYPE), true+gcc)
+    JVM_PRECOMPILED_HEADER_EXCLUDE := \
+        sharedRuntimeTrig.cpp \
+        sharedRuntimeTrans.cpp \
+        #
+  endif
+
   ifeq ($(TOOLCHAIN_TYPE), clang)
     JVM_PRECOMPILED_HEADER_EXCLUDE := \
-	sharedRuntimeTrig.cpp \
-	sharedRuntimeTrans.cpp \
+        sharedRuntimeTrig.cpp \
+        sharedRuntimeTrans.cpp \
         $(OPT_SPEED_SRC) \
-	#
+        #
   endif
```

Comment:
This looks to be in conflict with https://github.com/openjdk/jdk/pull/23268

Reply:
Ah, I missed that one. This seems like a more conservative fix, but I can go with the one in https://github.com/openjdk/jdk/pull/23268 as well and revert this.